cosmic-usage-ui/src/main/resources/public/app/main.js

```diff
@@ -27,6 +27,19 @@ const app = {
     generateReportButton: '#ui-generate-report-btn',
     printingHeadersContainer: '#ui-printing-headers',
     domainsTable: '#ui-domains-table',
+    domainsTableDomainHeadId: '#ui-domains-table-domain-head',
```

Comment:
These are all IDs, I would suggest to remove the `Id` part of the name of these attributes.